conceptual/Npgsql/connection-string-parameters.md

```diff
@@ -87,9 +95,12 @@ Application Name         | The optional application name parameter to be sent to
 Enlist                   | Whether to enlist in an ambient TransactionScope.                                                    | true
 Search Path              | Sets the schema search path.                                                                         |
 Client Encoding          | Gets or sets the client_encoding parameter.                                                          | PGCLIENTENCODING
+Encoding                 | Gets or sets the .NET encoding that will be used to encode/decode PostgreSQL string data.            | UTF8
 Timezone                 | Gets or sets the session timezone.                                                                   | PGTZ
 EF Template Database     | The database template to specify when creating a database in Entity Framework.                       | template1
+EF Admin Database        | The database admin to specify when creating and dropping a database in Entity Framework.             | template1
 Load Table Composites    | Load table composite type definitions, and not just free-standing composite types.                   | false
+ArrayNullabilityMode     | Configure the way arrays of value types are returned when requested as object instances. Possible values are: Never (arrays of value types are always returned as non-nullable arrays), Always (arrays of value types are always returned as nullable arrays) and PerInstance (the type of array that gets returned is determined at runtime).             | Never
```

Comment:
This is a mistake - it should be `Array Nullability Mode` (with spaces), we should correct this in the code...